components/basecomponents/LanguageSelect.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ButtonHTMLAttributes, forwardRef } from 'react';
+import ChevronRight from '../../public/icons/chevron-right.svg';
+
+interface LanguageSelect extends ButtonHTMLAttributes<HTMLButtonElement> {
+  active: boolean;
+  languages: [string, string];
```

Comment:
Can type this more narrowly as `[Language, Language]`

Reply:
Probably we should use CZ and UA (Language type is 'cs' | 'uk')  for Language Select component since CS and UK could be confusing for users. But I am open to discussion.

Reply:
Ah, I see, maybe it is better to stick with CZ/UA, even if it's not consistent with the URL. The localization codes will always be slightly confusing/inconsistent I guess. Maybe `Укр -> Česky` is a way out? Don't use any codes unless really necessary? Spell it out if you have the space? @petrillek